src/event.rs

```diff
@@ -90,6 +90,18 @@ pub struct AddressConsolidationNeeded {
     pub address: AddressWrapper,
 }
 
+#[cfg(any(feature = "ledger-nano", feature = "ledger-nano-simulator"))]
+/// Ledger generate address event data.
+#[derive(Clone, Debug, Getters, Serialize, Deserialize)]
+#[getset(get = "pub")]
+pub struct LedgerAddressGeneration {
+    #[serde(rename = "accountId")]
+    /// The associated account identifier.
+    pub account_id: String,
+    /// The transfer event type.
```

Comment:
is `addressData` the event type? The naming is kind of misleading

Reply:
The Event type is LedgerAddressGeneration or TransferProgressType, I will update to comment for the struct